lib/App/index.js

```diff
@@ -311,6 +308,29 @@ class App {
 		this.debug(`Validated successfully`);
 		
 	}
+
+    _checkZwaveForSetting(driver, setting) {
```

Comment:
What if `setting` is undefined (and driver for that matter)? Probably won't happen, but personally I like defensive programming:p

Reply:
I think that'd be overdoing it as it adds a bit of complexity to the code that doesn't necessarily have to be there. As this is called from within a for-loop over drivers and their settings and the parameters passed along are the currently being iterated values.

Reply:
I think that is a matter of taste then, but it would be easily solved (`_checkZwaveForSetting(driver, setting = {})`), anyway, was just meant as a tip, I see it won't cause real problems here.

Reply:
Fixed on the next line, added it to the check to see if setting.zwave exists!